scripts/etch-language-tests/maths/abs.etch

```diff
@@ -0,0 +1,23 @@
+function main()
```

Comment:
there are already several math tests in vm_modules/tests/unit/math.cpp (though not as extensive as this one) - so what is the correct place for tests? are we moving them all to scripts/etch-language-tests?